test/test_browser.py

```diff
@@ -5344,8 +5344,6 @@ def test_dlmalloc_3GB(self):
   })
   @requires_threads
```

Comment:
Why are no test changes needed for OPFS? Were we never testing it on the main thread?

Reply:
Right, we only ever test OPFS with `PROXY_TO_PTHREAD`.